src/main/resources/META-INF/plugin.xml

```diff
@@ -214,47 +214,44 @@
         </action>
 
         <action id="cody.command.Smell"
-                icon="/icons/chat/newChat.svg"
+                icon="/icons/chat/chat_command.svg"
                 class="com.sourcegraph.cody.chat.actions.SmellCommand">
             <keyboard-shortcut first-keystroke="control shift S"
                                keymap="Mac OS X 10.5+"/>
             <keyboard-shortcut first-keystroke="alt shift 2" keymap="$default"/>
             <add-to-group group-id="CodyEditorActions" anchor="last"/>
         </action>
 
-        <action id="cody.command.Test"
-                icon="/icons/chat/newChat.svg"
-                class="com.sourcegraph.cody.chat.actions.TestCommand">
-            <keyboard-shortcut first-keystroke="control shift T"
-                               keymap="Mac OS X 10.5+"/>
-            <keyboard-shortcut first-keystroke="alt shift 3" keymap="$default"/>
-            <add-to-group group-id="CodyEditorActions" anchor="last"/>
-        </action>
-
         <action id="cody.restartAgent"
                 class="com.sourcegraph.cody.agent.action.CodyAgentRestartAction"/>
         <group id="CodyChatActionsGroup">
             <reference ref="cody.newChat"/>
         </group>
 
         <!-- Inline editing actions -->
-        <action id="cody.editCodeAction"
-                class="com.sourcegraph.cody.edit.actions.EditCodeAction"
-                text="Edit Code"
-                description="Opens the Edit Code dialog">
+        <action id="cody.testCodeAction"
+                icon="/icons/edit/generate_test.svg"
```

Comment:
sadly, the latest snapshot of IntelliJ does not support the icons (yet?) 😢 

<img width="545" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/0b15ee71-b19a-42c7-887c-099bc6b91133">